forwards_paper.tex

```diff
@@ -318,6 +319,20 @@ \subsection*{Tables for succinct tree sequences}
 sequences~\citep{kelleher2016efficient}, normalised to remove redundancy
 and generalised to include a more general class of tree topologies.
 
```

Comment:
I think this is where I had just moved this paragraph *from*? heh. =)  I thought it got in the way of the flow here, and I'm less concerned about the Discussion being a bit of a random bag.  All we're doing here is connecting to the ARG literature, we are not really providing results (or methods); so shouldn't it be elsewhere?  I could try to shorten it and put it back in the introduction, alternatively.

Reply:
oh, ok. maybe this should go in the Discussion, next to the paragraph stating "The idea of storing genealogical information to speed up simulations is not new" because that is also about connections to other work?  Could have a `paragraph{Other methods of storing ancestry}` heading?

Reply:
Hmmm. Could we just stick this in somewhere in the introduction as literally one sentence (just citing Griffiths and one or two others)?

Reply:
will have a go as in #67